hbase-backup/src/test/java/org/apache/hadoop/hbase/backup/TestBackupUtils.java

```diff
@@ -87,21 +87,26 @@ public Path run() {
 
   @Test
   public void testFilesystemWalHostNameParsing() throws IOException {
-    String host = "localhost";
+    String host = "a-region-server.domain.com";
```

Comment:
I think we should try different patterns of server names here, like 'localhost', ip address, typical hostname, etc.

Reply:
ok, done.